tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot

```diff
@@ -66,7 +66,16 @@ JupyterLab Code Cell Error Output Should Not Be Visible
 
 Get JupyterLab Code Cell Error Text
   ${error_txt} =  Get Text  //div[contains(@class,"jp-OutputArea-output") and @data-mime-type="application/vnd.jupyter.stderr"]
-  [Return]
+  [Return]  ${error_txt}
+
+Run Git Repo And Return Last Cell Error Text As List
+  [Documentation]    It actually clones the git repo, runs it and then returns the last error
+  [Arguments]    ${REPO_URL}  ${NOTEBOOK_TO_RUN}
+  Run Keyword And Ignore Error    Clone Git Repository And Run    ${LINK_OF_GITHUB}    ${PATH_TO_FILE}
+  ${output} =    Get JupyterLab Code Cell Error Text
+  ${output} =    Split String    ${output}    \n\n
```

Comment:
this operation seems to be very specific to ODS-539 while this keyword appears to be a generic one. What do you think @pranavsuryaw @lugi0

Reply:
I think I should update it as
   `Run Keyword And Ignore Error    Clone Git Repository And Run    ${LINK_OF_GITHUB}    ${PATH_TO_FILE}

    ${output} =    Get JupyterLab Code Cell Error Text
    ${output} =    Split String    ${output}    \n\n`

Reply:
yes, I agree with you

Reply:
I am not sure what you mean @bdattoma ?

Reply:
@lugi0 sorry I might have explained badly. 
By reading the name of this keyword I feel like this is a generic keyword which aims to Clone a repo, Run one the notebook files in there, and return the Error text from the last cell in the notebook. 

While, the actual implementation return an elaboration of the "Error text from the last cell in the notebook" which seems to be something strictly related to this particular test case (which requires only the last line of the error.

Reply:
Oh, understood. Yes, I agree with you.
@pranavsuryaw could you remove lines 76 and 77 and return the full output? Then you can do the split and slice in your TC

Reply:
okay